### ISSUE TITLE
Make a Pijul version?

### ISSUE THREAD

**Issue by no-identd** [blinry/legit#5]
You know you want to.

**Comment by blinry** [comment]
You sent me down a very interesting rabbit hole by posting this! :D Are you actively using Pijul?

**Comment by no-identd** [comment]
No, because it ain't fast enough yet for the horrible abuse to which I wanna subject it, much worse than what you do here. :p I don't have a lot of time to type a reply here, so, I'll just hint at what I wanna do:

1. Pijul supports recursive patches (patches of patches, which I believe by now also includes patches which patch themselves.), which has interesting implications about the structure behind it all: https://discourse.pijul.org/t/patching-patches/212/19 see also: https://twitter.com/no_identd/status/1081363048231829510
2. https://discourse.pijul.org/t/ast-level-diffs-and-merges/187/
3. Instant messaging platforms annoy the hell out of me.
4. http://ict-arcfire.eu/index.php/rina/
5. https://twitter.com/no_identd/status/1140511614388621312
6. https://twitter.com/no_identd/status/1127982605381570562
7. https://twitter.com/no_identd/status/1127975588327772161
8. https://twitter.com/no_identd/status/1095970221758787584
9. https://twitter.com/disconcision/status/1093960049658073088 (Expand subthreads.)
